src/content/nl/2020/cms.md

```diff
@@ -0,0 +1,527 @@
+---
+part_number: III
+chapter_number: 15
+title: CMS
+description: CMS-hoofdstuk van de Web Almanac van 2020 over CMS-acceptatie, hoe CMS-suites worden gebouwd, gebruikerservaring van CMS-aangedreven websites en CMS-innovatie.
+authors: [alexdenning]
+reviewers: [sirjonathan, ernee, amedina]
+analysts: [GregBrimble, rviscomi]
+translators: [noah-vdv]
+alexdenning_bio: Alex Denning is de oprichter van <a href="https://getellipsis.com/">Ellipsis Marketing</a>, een marketingbureau voor WordPress-bedrijven. Alex is een WordPress Core-bijdrager en heeft geholpen bij het organiseren van <a href="https://london.wordcamp.org/">WordCamp London</a>.
```

Comment:
In #1781 and #1793 we added the ability to highlight a light as being in a different language to the text by adding the `hreflang` attribute to links.

```suggestion
alexdenning_bio: Alex Denning is de oprichter van <a hreflang="en" href="https://getellipsis.com/">Ellipsis Marketing</a>, een marketingbureau voor WordPress-bedrijven. Alex is een WordPress Core-bijdrager en heeft geholpen bij het organiseren van <a hreflang="en" href="https://london.wordcamp.org/">WordCamp London</a>.
```

This then looks like this:

![image](https://user-images.githubusercontent.com/10931297/102760304-397e3200-436d-11eb-9c5e-5f11cd069847.png)

Could you try to add these attributes to non-Dutch links from now on? Note likes including "en" like MDN and wikipedia links will automatically get this added so no need to do that for them.